src/main/java/com/iwaconsolti/league/service/PlayerService.java

```diff
@@ -1,4 +1,63 @@
 package com.iwaconsolti.league.service;
+import com.iwaconsolti.league.Config.PlayerConfig;
+import com.iwaconsolti.league.model.PlayerModel;
+import com.iwaconsolti.league.model.TeamModel;
+import org.springframework.stereotype.Service;
 
+import java.util.ArrayList;
+import java.util.List;
+
+@Service
 public class PlayerService {
+    private PlayerConfig playerConfig;
+
+    private final List<PlayerModel> playerList = new ArrayList<>();
+
+public PlayerService(PlayerConfig playerConfig){
```

Comment:
PlayerService(PlayerConfig playerConfig) constructor injects PlayerConfig, it does not have @Autowired.